$NetBSD: patch-aa,v 1.3 2009/11/01 02:19:39 obache Exp $

Since the 'roman' module is unconditionally present in the PLIST,
it must be unconditionally built and installed.
(when building the py-docutils package while already having it installed,
setup.py excluded the module, causing PLIST breakage with USE_DESTDIR=yes)

--- setup.py.orig	2009-10-05 19:08:10.000000000 +0000
+++ setup.py
@@ -174,13 +174,12 @@ classifiers = [
 """Trove classifiers for the Distutils "register" command;
 Python 2.3 and up."""
 
-extra_modules = [('roman', '1.4', ['toRoman', 'fromRoman',
-                                   'InvalidRomanNumeralError'])]
+extra_modules = []
 """Third-party modules to install if they're not already present.
 List of (module name, minimum __version__ string, [attribute names])."""
 
 def get_extras():
-    extras = []
+    extras = ['roman']
     for module_name, version, attributes in extra_modules:
         try:
             module = __import__(module_name)
